content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -29,22 +29,21 @@ mitigate some future vulnerabilities too.
 <!-- body -->
 ## {{% heading "prerequisites" %}}
 
-{{% thirdparty-content single="true" %}}
-<!-- if adding another runtime in the future, omit the single setting -->
+{{% thirdparty-content %}}
 
 This is a Linux only feature. In addition, support is needed in the 
 {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
 to use this feature with Kubernetes stateless pods:
 
-* CRI-O: v1.25 has support for user namespaces.
+* CRI-O: version 1.25 (and later) supports user namespaces for containers.
 
-* containerd: support is planned for the 1.7 release. See containerd
-  issue [#7063][containerd-userns-issue] for more details.
+* containerd: version 1.7 supports user namespaces for containers, compatible
+  with Kubernetes {{< skew currentVersion >}}. If you are running a different
```

Comment:
This will break if we apply this to 1.25. This will point to kubernetes 1.24, IIUC, which simply doesn't support user namespaces

Reply:
Yep, I don't recommend backporting _this_ line to the v1.25 docs

Reply:
IMHO it will be simpler if we just use the versions, as they are not relative to any k8s version. In fact, here just expands to k8s 1.26, but this is true with k8s 1.25 too.

Reply:
I'd prefer calling out the specific version in this context.
We say "the WWII ended in 1945" instead of "in {{ current year }}, the WWII has ended." with a constant refreshing of the `{{ current year }}`.

Reply:
Historical events aren't quite the right analogy. It's about a commutative relationship between (releases of) two things. The containerd docs might want to express the reverse relationship for example.

The text I've proposed for v1.27 also uses a skew shortcode (for a different reason).

I guess if we're sure that v1.7 of containerd is compatible specifically with v1.26 of Kubernetes and no other minor version of this project, we _should_ say exactly that (and keep the “If you are running a different version of Kubernetes, check the documentation for that Kubernetes release.”)

However, this is an unusual case because of when we're making the change, ie after the code freeze for v1.27

Reply:
@sftim what change is after code freeze? This doc PR?

First, this doc isn't for 1.27. This is for 1.25 and 1.26.

Secondly, that statement is true for 1.25 and 1.26, I don't understand why you insist in mentioning only one release.

And yes, we re-worked the implementation in 1.27 and what we need other requirements from the runtime. That is why it is not recommended to use 1.7 for userns with k8s >= 1.27.

But to return to more practical matter, how do you want me to phrase it here? So we can merge this PR

Reply:
I commented: https://github.com/kubernetes/website/pull/40264/files#r1149286339

Reply:
@sftim that link is not working (github is having hiccups, maybe is due to that). Can you please explain here again? Or a suggestion is even better ;)